conda/fetch.py

```diff
@@ -283,6 +283,17 @@ def fetch_pkg(info, dst_dir=None, session=None):
     path = join(dst_dir, fn)
 
     download(url, path, session=session, md5=info['md5'], urlstxt=True)
+    if info.get('sig'):
+        from conda.signature import verify
+
+        fn2 = fn + '.sig'
+        url = (info['channel'] if info['sig'] == '.' else
+               info['sig'].rstrip('/') + '/') + fn2
+        log.debug("url=%r" % url)
```

Comment:
This is a duplicate of the `log.debug()` call in line 282 above.  That seems confusing when actually trying to debug this problem.  Maybe this should be `log.debug("signature url=%r" % url)`?